libs/shared/src/commonProtocol/chainConfig.ts

```diff
@@ -39,4 +43,24 @@ export const factoryContracts: {
     communityStake: '0xcc752fd15A7Dd0d5301b6A626316E7211352Cf62',
     chainId: 8453,
   },
+  [ValidChains.Linea]: {
+    factory: '0xe3ae9569f4523161742414480f87967e991741bd',
+    communityStake: '0xcc752fd15a7dd0d5301b6a626316e7211352cf62',
+    chainId: 59144,
+  },
+  [ValidChains.Optimism]: {
+    factory: '0xe3ae9569f4523161742414480f87967e991741bd',
+    communityStake: '0xcc752fd15a7dd0d5301b6a626316e7211352cf62',
+    chainId: 10,
+  },
+  [ValidChains.Mainnet]: {
+    factory: '0x90aa47bf6e754f69ee53f05b5187b320e3118b0f',
+    communityStake: '0x9ed281e62db1b1d98af90106974891a4c1ca3a47',
+    chainId: 1,
+  },
+  [ValidChains.Arbitrum]: {
+    factory: '0xE3AE9569f4523161742414480f87967e991741bd',
+    communityStake: '0xcc752fd15A7Dd0d5301b6A626316E7211352Cf62',
+    chainId: 42161,
+  },
```

Comment:
@timolegros is the warning about EvmEventSources still valid? If so will include that here as well

Reply:
Yes. If these are newly deployed contracts that we want to support then we need to listen for events on them and that only happens if new EvmEventSources records are created.

Reply:
Got it will add those migrations to this PR

Reply:
added the migration, let me know if I missed anything